src/client.js

```diff
@@ -13,6 +13,9 @@ class Client {
     this.socket.on("close", function() {
       process.exit();
     });
+    this.socket.on("error", function() {
+      // Prevent Unhandled Exception resulting from TCP Error
```

Comment:
I think that would be a good idea to at least log the error here using `console.error` or an alternative.

Reply:
That's what I told him before he submitted this, he wouldn't listen 🙄